### PR TITLE
DOCS-415: median* functions

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation

Added English and Russian descriptions for the median* functions. Solved #7340.

### CODE REVIEW

File: docs/en/query_language/agg_functions/reference.md [hunk 1]
```diff
@@ -908,32 +908,456 @@ SELECT quantileTiming(0.5)(number / 2) FROM numbers(10)
...
+Alias for [quantile(0.5)(x)](#agg_function-quantile).
+
+**Parameters** 
+
+- `x` — Number. Expression resulting in numeric [data types](../../data_types/index.md#data_types), [Date](../../data_types/date.md) or [DateTime](../../data_types/datetime.md).
+
+**Returned value**
+
+- Middle value of a sorted resampled sequence if it contains an odd count of numbers.
+- Arithmetic mean of two middle values of a sorted resampled sequence if it contains an even count of numbers. The function doesn't round the result.
```
Comment (alexey-milovidov) [inline docs/en/query_language/agg_functions/reference.md]: > The function doesn't round the result.

Don't understand. Clarification needed.

File: docs/en/query_language/agg_functions/reference.md [hunk 2]
```diff
@@ -908,32 +908,456 @@ SELECT quantileTiming(0.5)(number / 2) FROM numbers(10)
...
 
 The result depends on the order of running the query, and is nondeterministic.
 
-## median(x)
+## median {#median}
 
-All the quantile functions have corresponding median functions: `median`, `medianDeterministic`, `medianTiming`, `medianTimingWeighted`, `medianExact`, `medianExactWeighted`, `medianTDigest`. They are synonyms and their behavior is identical.
+Computes an approximate [median](https://en.wikipedia.org/wiki/Median) of a numeric data sequence.
+
+This function applies [reservoir sampling](https://en.wikipedia.org/wiki/Reservoir_sampling) with a reservoir size up to 8192. The function uses a random number generator for reservoir sampling, so its result is non-detetministic and has low-accuracy. To get an exact median, use the [medianExact](#medianexact) function.
```
Comment (alexey-milovidov) [inline docs/en/query_language/agg_functions/reference.md]: What does it mean "low accuracy"?

File: docs/en/query_language/agg_functions/reference.md [hunk 3]
```diff
@@ -895,47 +1207,197 @@ Type: `Float32`.
...
-## quantileExact(level)(x)
 
-Computes the quantile of 'level' exactly. To do this, all the passed values ​​are combined into an array, which is then partially sorted. Therefore, the function consumes O(n) memory, where 'n' is the number of values that were passed. However, for a small number of values, the function is very effective.
+## quantileTDigest {#quantiletdigest}
 
-## quantileExactWeighted(level)(x, weight)
+Computes an approximate [quantile](https://en.wikipedia.org/wiki/Quantile) of a numeric data sequence using the [t-digest](https://github.com/tdunning/t-digest/blob/master/docs/t-digest-paper/histo.pdf) algorithm.
 
-Computes the quantile of 'level' exactly. In addition, each value is counted with its weight, as if it is present 'weight' times. The arguments of the function can be considered as histograms, where the value 'x' corresponds to a histogram "column" of the height 'weight', and the function itself can be considered as a summation of histograms.
+The maximum error is 1%.  Memory consumption is `log(n)`, where `n` is a number of values. The result depends on the order of running the query, and is nondeterministic.
```
Comment (qoega) [inline docs/en/query_language/agg_functions/reference.md]: `The maximum error is 1%.` - error is not defined. We need to explain this better.
PS. In previous doc version it is the same.

File: docs/en/query_language/agg_functions/reference.md [hunk 4]
```diff
@@ -895,47 +1207,197 @@ Type: `Float32`.
...
+```
+
+**See Also**
+
+- [median](#median)
+- [quantiles](#quantiles)
+
+## quantileTDigestWeighted {#quantiletdigestweighted}
+
+Computes an approximate [quantile](https://en.wikipedia.org/wiki/Quantile) of a numeric data sequence using the [t-digest](https://github.com/tdunning/t-digest/blob/master/docs/t-digest-paper/histo.pdf) algorithm. The function takes into account the weight of each sequence number. The maximum error is 1%.  Memory consumption is `log(n)`, where `n` is a number of values.
```
Comment (qoega) [inline docs/en/query_language/agg_functions/reference.md]: `sequence number` or `sequence member`? There are two variants in this PR

**Pull request merged**